docs/DeveloperGuide.adoc

```diff
@@ -273,94 +275,121 @@ Refer to the guide <<Testing#, here>>.
 == Dev Ops
 
 Refer to the guide <<DevOps#, here>>.
+////
 
 [appendix]
 == Product Scope
+=== Target User Profile
+* Health-conscious people watching their diet
+* Prefer desktop applications over mobile apps
+* Comfortable enough to work with text command-based interfaces
 
-*Target user profile*:
-
-* has a need to manage a significant number of contacts
-* prefer desktop apps over other types
-* can type fast
-* prefers typing over mouse input
-* is reasonably comfortable using CLI apps
-
-*Value proposition*: manage contacts faster than a typical mouse/GUI driven app
+=== Value Proposition
+The app is able to help the user keep track of their calorie consumption. This way, they can ensure that they keep to their own diet plans.
 
 [appendix]
 == User Stories
 
-Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (unlikely to have) - `*`
-
-[width="59%",cols="22%,<23%,<25%,<30%",options="header",]
+[width="59%",cols="4*",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
-
-|`* * *` |user |add a new person |
-
-|`* * *` |user |delete a person |remove entries that I no longer need
+|&#11088;&#11088;&#11088;
+|new user
+|see usage instructions
+|refer to instructions when I forget how to use the App.
+
+|&#11088;&#11088;&#11088;
+|user
+|create a new meal entry
+|keep track of what I ate today.
+
+|&#11088;&#11088;&#11088;
+|user
+|see how many calories I have left in today's budget
+|know what I can eat later, and stay in budget.
+
+|&#11088;&#11088;&#11088;
+|user
+|be alerted if I missed a meal
+|be reminded to stick to my meal plans.
+
+|&#11088;&#11088;&#11088;
+|user
+|change my daily calorie budget
+|better reflect my new lifestyle requirements.
```

Comment:
(Reviewing for tutorial exercise) Can the "So that I can..." part for this can be more specific?